### PR TITLE
feat(formActions): form actions component

### PR DESCRIPTION
**Pull request by Anjmao**

### CODE REVIEW

File: src/form-actions/docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+</form-actions>
+```
+
+### Form actions bar settings ###
+ * `append`
+ 	:
+ 	By default form actions bar is fixed to window bottom. If element is specified it will be fixed to that element.
+
+```html
+<form-actions append=".side-panel">
```
Comment (varnastadeus) [inline src/form-actions/docs/readme.md]: apend-to in that case

File: template/form-actions/form-actions.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div ng-show="form.$dirty" class="fixed-form-actions-bar">
```
Comment (varnastadeus) [inline template/form-actions/form-actions.html]: how about overlay when it slides out?

File: src/form-actions/form-actions.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('dd.ui.form-actions', [])
+        .directive('formActions', formActions);
```
Comment (varnastadeus) [inline src/form-actions/form-actions.js]: why not component? :d

Comment (Anjmao) [inline src/form-actions/form-actions.js]: In this case we need to upgrade dd-ui and admin angular version to 1.5 and this needs some time.

File: template/form-actions/form-actions.html (hunk 2 above)
Comment (Anjmao) [inline template/form-actions/form-actions.html]: Sorry ?

**Pull request merged**